.gitignore

```diff
@@ -19,3 +19,6 @@ confdoc.ser
 .buildpath
 .project
 .settings/
+
+.idea
+vendor
```

Comment:
Can you please remove those lines as they are not related to the feature itself?

Reply:
the lines are removed